src/kubeEvents.ts

```diff
@@ -0,0 +1,25 @@
+import { Kubectl } from './kubectl';
+import * as kubectlUtils from './kubectlUtils';
+import * as explorer from './explorer';
+
+export enum EventDisplayMode {
+    Show,
+    Follow
+}
+
+export async function getEvents(kubectl: Kubectl, displayMode: EventDisplayMode, explorerNode?: explorer.ResourceNode) {
+    let eventsNS;
+
+    if (explorerNode) {
+        eventsNS = explorerNode.id;
+    } else {
+        eventsNS = await kubectlUtils.currentNamespace(kubectl);
+    }
+
+    let cmd = ` get events --namespace ${eventsNS}`;
```

Comment:
You don't need the leading space here - the `kubectl` methods handle this for you.